src/components/TerminalOutput/TerminalOutput.js

```diff
@@ -79,7 +79,7 @@ class TerminalOutput extends PureComponent<Props> {
           >
             <FillButton
               size="xsmall"
-              colors={[COLORS.red[700], COLORS.red[500]]}
+              colors={[COLORS.pink[300], COLORS.red[500]]}
```

Comment:
Should this be a defined color set? Perhaps `import { destructive } from '.../button-colors'`?

Reply:
Indeed, I think we should move away from referencing colors directly and use semantic variables but that should probably be another PR as it’s a pretty big change

Reply:
+1 I really like this pattern

Reply:
++, this is something I should have done earlier. It speaks to a broader issue, that I don't really have a design system, it was all very figure-it-out-as-I-go. So I've just been picking colours based on aesthetics, which in addition to leading to inconsistencies, makes it really hard for other folks to contribute to the UI.

But yeah, that's a wayyy broader issue. If someone wants to start moving us towards semantic colours, that'd be awesome, but it should be its own issue with several PRs.